### PR TITLE
Fix docker file

### PR DESCRIPTION
**Pull request by sladyn98**
1) Fixed typo in frontend Dockerfile

2) Adds a dockerignore for the backend thus reducing the size

### CODE REVIEW

**Comment by martinda** [comment]
Looks like a good step forward to me. It does not fix everything. I still do not understand why I get nothing at http://localhost:8080 when I run the frontend and the backend.

File: Dockerfile [hunk 1]
```diff
@@ -11,10 +11,10 @@ VOLUME /tmp
 EXPOSE 8080
 
 # The application's jar file
-ARG JAR_FILE=target/custom-distribution-service-0.0.1.jar
+ARG JAR_FILE=target/*.jar
 
 # Add the application's jar to the container
-ADD ${JAR_FILE} custom-distribution-service.jar
+ADD ${JAR_FILE} app.jar
```
Comment (kwhetstone) [inline Dockerfile:17]: Should this be app?

**Comment by kwhetstone** [comment]
@sladyn98 looks like this needs to be rebased in order to be merged.  Will rereview when this is rebased.

**Comment by martinda** [comment]
Please rebase on latest master.

**Comment by sladyn98** [comment]
Rebased

**Pull request merged**